plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -1283,8 +1283,11 @@ public void renameColumn(ConnectorSession session, ConnectorTableHandle tableHan
     }
 
     @Override
-    public void dropColumn(ConnectorSession session, ConnectorTableHandle tableHandle, ColumnHandle column)
+    public void dropColumn(ConnectorSession session, ConnectorTableHandle tableHandle, ColumnHandle column, List<String> field)
     {
+        if (!field.isEmpty()) {
+            throw new TrinoException(NOT_SUPPORTED, "This connector does not support dropping fields from row types");
```

Comment:
Skip updates to connectors other than Iceberg.
This will help ensure the deprecation-related routing works correctly.